### ISSUE TITLE
Candle Chart to Line Chart zoom out transition

### ISSUE THREAD

**Issue by FisherWL** [syncfusion/flutter-widgets#676]
a common feature of a stock app on a candlestick chart page is that:
when zooming out the chart to get a longer time span,
the chart transit from candle to line from a certain zooming factor.
(vice versa)

how can I implement this with syncfusion?
thank you for your time!

**Comment by FisherWL** [comment]
it seems the answer is  onZooming: (ZoomPanArgs args)

**Comment by Yuvaraj-Gajaraj** [comment]
Hi Team,

We have created a sample that converts the series type from candle to line series when the zoom factor value goes less than 0.5, by default the zoom factor ranges from 0.1 to 1.0. We have achieved this with the help of the [onZooming](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCartesianChart/onZooming.html) callback which is used to get the current zoom factor of the chart, here we have changed the series type based on the zoom factor. If the current factor is less than 0.5 it will change to line series, if greater than 0.5 it converts to candle series. This can be changed based on your scenario, and we have attached the sample below for your reference.

UG: https://help.syncfusion.com/flutter/cartesian-charts/callbacks#onzooming

Sample: [i375666.zip](https://github.com/syncfusion/flutter-widgets/files/8505809/i375666.zip)


Regards,
Yuvaraj.
